lib/types.d.ts

```diff
@@ -295,14 +301,19 @@ export interface LiteralDefNumber extends LiteralDefBase {
 
 export interface LiteralDefBigInt extends LiteralDefBase {
   kind: "bigInt";
-  number: bigint;
+  string: string;
```

Comment:
Is this change on purpose?

Reply:
Yeah, it was a mistake on my part, and I didn't notice it until I was making the other changes.  Silly me forgot there is no way to serialize a bigint to JSON as a bigint.